.github/CODEOWNERS

```diff
@@ -58,7 +58,7 @@ packages/styling/src/interfaces/ @phkuo
 packages/styling/src/styles/ @phkuo
 # packages/utilities/
 packages/utilities/positioning/ @joschect
-packages/date-time @OfficeDev/uifabric-calendar @jh2he @evlevy
```

Comment:
@evlevy is still quite active in working on calendar

Reply:
Oh, didn't realize that because a search only showed a single contribution https://github.com/OfficeDev/office-ui-fabric-react/pulls?q=evlevy

Reply:
Yes don't remove me. @jh2he can go. He's not active on this project.

Reply:
Fixed